recipes/starshot_card_block_type/config/field.field.block_content.card.field_card_media.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.card
+    - field.storage.block_content.field_card_media
+    - media.type.image
+    - media.type.remote_video
+    - media.type.video
+id: block_content.card.field_card_media
+field_name: field_card_media
```

Comment:
Same comment as with link... Should it be reused? and media is singular and plural. The intention seems to be a single media item for displaying an image or a video. I often choose "hero" for this which is a very common and meaningful way to describe a single media instance that is canonical for an entity.

Reply:
I'm okay with renaming this to `field_hero`.

Reply:
I don't know that hero is correct here given there will be multiple cards in a card grid

Reply:
Echoing https://github.com/phenaproxima/starshot-prototype/pull/76#discussion_r1602382429, I'd recommend we not get too deep into the weeds of how the fields are named -- at least not in this PR.